examples/tests/sdk_tests/util/spawn.test.w

```diff
@@ -0,0 +1,57 @@
+bring util;
+bring expect;
+
+let assertThrows = inflight (expected: str, block: (): void) => {
+  let var error = false;
+  try {
+    block();
+    } catch actual {
+      expect.equal(actual, expected);
+      error = true;
+    }
+    expect.equal(error, true);
+};
+
+
+test "spawn() with successful execution" {
+  let program = "echo";
+  let args = ["Hello, Wing!"];
```

Comment:
Can you try with no arguments too?

Reply:
You mean with empty (`""`) arguments? Because `args` is a required parameter in `spawn()`.

Reply:
yes, an empty array